docs/enableexperimental.md

```diff
@@ -0,0 +1,36 @@
+# How to enable experimental features
+
+Fleet supports experimental features that are disabled by default and that can be enabled by the user.
+
+Enabling/disabling experimental feaures is done usign extra environment variales that are available when deploying rancher/fleet.
```

Comment:
```suggestion
Enabling/disabling experimental features is done using extra environment variables that are available when deploying `rancher/fleet`.
```

Apart from typo fixes, this adds backticks around `rancher/fleet` to clarify that we're talking about a chart. Unless the intended meaning was rather something like "Rancher or Fleet", as environment variables would be available in both cases...?